source/includes/administration/_organizations.md

```diff
@@ -233,3 +233,209 @@ curl -X DELETE "https://cloudmc_endpoint/v1/organizations/03bc22bd-adc4-46b8-988
 ```
 
 Returns an HTTP status code 204, with an empty response body.
+
+<!-------------------- GET VERIFIED DOMAINS -------------------->
+### Get verified domains
+`GET /organizations/:organization_id/verified_domains`
+
+Get a list of all verified domains on the specified organization.
+
+```shell
+# Retrieve all verified domains
+curl "https://cloudmc_endpoint/v1/organizations/87895f43-51c1-43cc-b987-7e301bf5b86a/verified_domains" \
+   -H "MC-Api-Key: your_api_key"
+```
+> The above command returns a JSON structured like this:
+
+```json
+{
+  "data": [
+    {
+      "lastCheckedDate": "2020-07-31T19:14:36Z",
+      "createdDate": "2020-07-31T19:12:55Z",
+      "domain": "www.aGoodFakeDomain.com",
```

Comment:
i'll change it to never return `www.` and `http://`. Could you remove please 🙏

Reply:
9a27d69